api/docker-entrypoint.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
```

Comment:
Any reason not to have this stop on error?
```suggestion
set -e
```